File: B_Not_Dividing.c
# include <stdio.h>

int main(){
    int t;
    scanf("%d", &t);

    while (t--){
        int n;
        scanf("%d", &n);
        int num[n];
        for (int i = 0; i < n; i++){
            scanf("%d", &num[i]);
            if (num[i] == 1){
                num[i]+=1;
            }
        }
        for (int i = 0; i < n - 1; i++){
            if(num[i+1] % num[i] == 0){
                num[i+1] += 1;
            }
        }
        for (int i = 0; i < n; i++)
        {
            printf("%d ", num[i]);
        }
        printf("\n");
                
    }
    return 0;
    
}

File: B_Not_Dividing.py
t = int(input())

for _ in range(t):
    n = int(input())
    num = list(map(int, input().split()))

    for i in range(n):
        if num[i] == 1:
            num[i] = 2

    for i in range(n - 1):
        if num[i + 1] % num[i] == 0:
            num[i + 1] += 1

    print(*num)